miqa/core/tasks.py

```diff
@@ -374,5 +376,8 @@ def perform_export(project_id: Optional[str]):
                     f'{frame_object.scan.name} not exported; this scan was uploaded, not imported.'
                 )
     export_df = pandas.DataFrame(data, columns=IMPORT_CSV_COLUMNS)
-    export_df.to_csv(export_path, index=False)
+    try:
+        export_df.to_csv(export_path, index=False)
+    except (PermissionError):
```

Comment:
```suggestion
    except PermissionError:
```

Reply:
Changed in [2d8fffb](https://github.com/OpenImaging/miqa/pull/578/commits/2d8fffbf2190793b4df9d619013ce156a0d31609)